Sources/AppStoreConnectCLI/Commands/BetaGroups/BetaGroupCommand.swift

```diff
@@ -11,7 +11,9 @@ struct BetaGroupCommand: ParsableCommand {
         one or more builds.
         """,
         subcommands: [
-            CreateBetaGroupCommand.self
-        ]
+            CreateBetaGroupCommand.self,
+            ListBetaGroupsCommand.self
+        ],
+        defaultSubcommand: ListBetaGroupsCommand.self
```

Comment:
we used to place stuff related to Test Flight under `AppStoreConnectCLI/Commands/TestFlight/` folder. you might think it's a bit too deep for user to use?

Reply:
It depends if we want the folder structure to reflect the API grouping or the command grouping.
Betagroup is a top level command: `asc betagroup` so that was my reasoning

Reply:
Beta Groups only apply to TestFlight so my thoughts were that we would have subcommands under testflight for betagroups etc.

Reply:
This is why #99 lists has the command as `asc testflight betagroup list`

Reply:
Haha, I somehow misread it 🤦